nilearn/decomposition/_subject_pca.py

```diff
@@ -0,0 +1,330 @@
+"""
```

Comment:
I don't think this is a good idea if we do not want to support `PCAMultiNiftiMasker`, which is really an internal tool.